packages/gasket-cli/package.json

```diff
@@ -69,6 +69,7 @@
     "@oclif/plugin-help": "^5.1.10",
     "@oclif/plugin-not-found": "^2.2.4",
     "@oclif/plugin-warn-if-update-available": "^2.0.4",
+    "browserslist-config-godaddy": "^3.0.1",
```

Comment:
Why is this needed? This appears to be an internal-only package that should not be a dependency of the open source packages. CI is failing to find it as a result:
https://github.com/godaddy/gasket/runs/7883994294?check_suite_focus=true#step:4:26